src/main/java/org/graylog2/syslog4j/impl/unix/socket/UnixSocketSyslog.java

```diff
@@ -39,6 +40,12 @@ public void setSunPath(String sunPath) {
             System.arraycopy(sunPath.getBytes(), 0, this.sun_path, 0, sunPath.length());
             System.arraycopy(ZERO_BYTE, 0, this.sun_path, sunPath.length(), 1);
         }
+
+		@Override
+		protected List getFieldOrder() {

```

Comment:
Why exactly does this method in `Structure` need to be overridden? And shouldn't it return an empty list (e. g. `Collections.emptyList()` instead of `null` if it absolutely has to be a no-op?